modules/system/models/mailbrandsetting/custom.less

```diff
@@ -75,6 +74,9 @@ code {
     color: @text-color;
     font-size: 16px;
     line-height: 1.5em;
+    -ms-word-break: break-all;
+    word-break: break-word;
```

Comment:
@vevers Is this necessary with the `break-all` rule directly underneath?

Reply:
I kept the original structure of using a combination of `-ms-word-break: break-all; word-break: break-all; word-break: break-word;` as used in the body rule intact, assuming it was required for some weird edge case or specific email clients. 

Just tried simplifying the rules as suggested, tested it again in Litmus and didn't see any problems.